packages/patternfly-4/react-core/src/components/Nav/Nav.js

```diff
@@ -49,8 +49,14 @@ class Nav extends React.Component {
     });
   }
 
+  // Determine if it's a tertiary nav or not (tertiary needs a different aria-label)
+  determineLabel() {
+    const isTertiary = React.Children.map(this.props.children, child => child.props.variant === 'tertiary');
```

Comment:
Why do you need to go through all the children here if you are only looking at the variant from the first child.
Can you just look at the first child and maybe do this inline rather than having a function?